UnityProject/Assets/Scripts/Items/Food/Edible.cs

```diff
@@ -112,38 +117,34 @@ public override void TryConsume(GameObject feederGO, GameObject eaterGO)
 
 			if (feeder != eater) //If you're feeding it to someone else.
 			{
-				//Wait 3 seconds before you can feed
 				StandardProgressAction.Create(ProgressConfig, () =>
 				{
 					ConsumableTextUtils.SendGenericForceFeedMessage(feeder, eater, eaterHungerState, Name, "eat");
 					Eat(eater, feeder);
-				}).ServerStartProgress(eater.RegisterPlayer, 3f, feeder.gameObject);
+				}).ServerStartProgress(eater.RegisterPlayer, forceFeedTime, feeder.gameObject);
 				return;
 			}
-			else
-			{
-				Eat(eater, feeder);
-			}
+			StandardProgressAction.Create(ProgressConfig, () =>
+				{
+					Eat(eater, feeder);
+				}).ServerStartProgress(eater.RegisterPlayer, consumeTime, feeder.gameObject);
 		}
 
 		protected virtual void Eat(PlayerScript eater, PlayerScript feeder)
 		{
 			//TODO: Reimplement metabolism.
-			AudioSourceParameters eatSoundParameters = new AudioSourceParameters(pitch: RandomPitch);
-			SoundManager.PlayNetworkedAtPos(sound, eater.WorldPos, eatSoundParameters, sourceObj: eater.gameObject);
-
-			var Stomachs = eater.playerHealth.GetStomachs();
-			if (Stomachs.Count == 0)
+			var stomachs = eater.playerHealth.GetStomachs();
+			if (stomachs.Count == 0)
 			{
 				//No stomachs?!
```

Comment:
I feel like there is a message missing here to tell the player they can't eat but I can't think of aynthing to say that would hint at a missing stomach

Reply:
Something like "You try to eat, but the food won't go down!" might work.

Also maybe vomit when you do that? Not sure if that would make sense to show you spitting the food out. Though if this happens you would probably still want to have the chewing sound and take some of the food away for it to make more sense.

Reply:
okay. I will merge this for now. We will have to discuss it further, but I want this merged to test the build candidate for this Saturday and this isn't really a critical issue.